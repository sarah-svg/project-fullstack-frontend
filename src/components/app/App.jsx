import React from 'react';
import {
  Switch,
  Route,
  Link
} from 'react-router-dom';
import MultRes from './Mult-Res';

import Map from './Map';
// import Image from './Image';
// import UHH from './UHH';
// import AddMapp from '../AddMapp';




export default function App() {
  return <>
    <Link to="/map">Link to add your map image and burn pile sizes</Link>

    <br></br>

    <Link to="/">Home Page</Link>

    <br></br>
    <Link to="/all">Get all of your maps</Link>
    <Link to="/image">Seeing if this image hits my database</Link>
    <Switch>
      <Route exact path="/map" component={Map}/>
      <Route exact path="/all" component={MultRes} />
    </Switch>

  </>;
}
