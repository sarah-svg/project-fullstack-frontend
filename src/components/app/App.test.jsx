import React from 'react';
import { render, cleanup } from '@testing-library/react';
import App from './App';
import { MemoryRouter } from 'react-router';

describe('App component', () => {
  afterEach(() => cleanup());
  it('renders App', () => {
    const { asFragment } = render(<MemoryRouter><App /></MemoryRouter>);
    expect(asFragment()).toMatchSnapshot();
  });
});
