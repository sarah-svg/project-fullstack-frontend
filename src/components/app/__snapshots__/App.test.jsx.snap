// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders App 1`] = `
<DocumentFragment>
  <a
    href="/map"
  >
    Link to add your map image and burn pile sizes
  </a>
  <br />
  <a
    href="/"
  >
    Home Page
  </a>
  <br />
  <a
    href="/all"
  >
    Get all of your maps
  </a>
</DocumentFragment>
`;
